pkg/standalone/run.go

```diff
@@ -73,17 +74,28 @@ func getDaprCommand(appID string, daprHTTPPort int, daprGRPCPort int, appPort in
 	}
 
 	daprCMD := binaryFilePath(defaultDaprBinPath(), "daprd")
-	metricsPort, err := freeport.GetFreePort()
-	if err != nil {
-		return nil, -1, -1, -1, err
-	}
 
-	args := []string{"--app-id", appID, "--dapr-http-port", fmt.Sprintf("%v", daprHTTPPort), "--dapr-grpc-port", fmt.Sprintf("%v", daprGRPCPort), "--log-level", logLevel, "--app-max-concurrency", fmt.Sprintf("%v", maxConcurrency), "--app-protocol", protocol, "--metrics-port", fmt.Sprintf("%v", metricsPort), "--components-path", componentsPath}
+	args := []string{
+		"--app-id", appID,
+		"--dapr-http-port", fmt.Sprintf("%v", daprHTTPPort),
```

Comment:
```suggestion
		"--dapr-http-port", strconv.Itoa(daprHTTPPort),
```
I think `fmt.Sprintf()` is both longer, less efficient, and more error-prone (ignore types). It's not the purpose of this PR but probably nice to include the clean up here. Same with other occurences.